Java 10, Java 11-EA builds failing due to EBR plugin

The Java 10 and Java 11-EA builds are failing due to API compatibility issue in the EBR plugin while creating OSGi artifacts.

Example log: https://api.travis-ci.org/v3/job/394162025/log.txt

```
21:19:47:456 [ERROR] Failed to execute goal org.eclipse.ebr:ebr-maven-plugin:1.0.0-SNAPSHOT:bundle (default-bundle) on project org.eclipse.collections: Execution default-bundle of goal org.eclipse.ebr:ebr-maven-plugin:1.0.0-SNAPSHOT:bundle failed: An API incompatibility was encountered while executing org.eclipse.ebr:ebr-maven-plugin:1.0.0-SNAPSHOT:bundle: java.lang.ExceptionInInitializerError: null
21:19:47:456 [ERROR] -----------------------------------------------------
21:19:47:457 [ERROR] realm =    extension>org.eclipse.ebr:ebr-maven-plugin:1.0.0-SNAPSHOT
```

@guw @ujhelyiz Can either of you please help?

@guw @ujhelyiz I ran the build with -e, cannot run in debug (-X) as the logs exceed Travis limit. Here is the link to log: https://api.travis-ci.org/v3/job/395048845/log.txt

```
Number of foreign imports: 1
import: Entry[import  from realm ClassRealm[maven.api, parent: null]]

-----------------------------------------------------

	at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:181)
	... 21 more
Caused by: java.lang.ExceptionInInitializerError
	at org.eclipse.ebr.maven.BundleMojo.assembleJar(BundleMojo.java:259)
	at org.eclipse.ebr.maven.BundleMojo.createPluginJar(BundleMojo.java:430)
	at org.eclipse.ebr.maven.BundleMojo.buildBundle(BundleMojo.java:365)
	at org.eclipse.ebr.maven.BundleMojo.execute(BundleMojo.java:462)
	at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:134)
	... 21 more
Caused by: java.lang.ArrayIndexOutOfBoundsException: 1
	at org.codehaus.plexus.archiver.zip.AbstractZipArchiver.<clinit>(AbstractZipArchiver.java:118)
	... 26 more
```

I will run the build in debug mode in my local machine later today, but wanted to give this info, in case it helps isolate the issue. Looking at the ArrayIndexOutOfBounds, I wonder is the ebr plugin compatible for Java 10, Java 11? I recon it might be because the version numbers have become double digit?

--- a/p2-repository/pom.xml
+++ b/p2-repository/pom.xml
@@ -49,8 +49,8 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 
         <!-- versions used for plug-ins below -->
-        <tycho-version>1.0.0</tycho-version>
-        <tycho-extras-version>1.0.0</tycho-extras-version>
+        <tycho-version>1.2.0</tycho-version>
+        <tycho-extras-version>1.2.0</tycho-extras-version>
 
         <!-- versions of the EBR plug-in to use -->
         <ebr-version>1.0.0-SNAPSHOT</ebr-version>

